test/unit/factories/Axis.mocha.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path='../test.mocha.ts' />
+
+describe('n3Charts.Factory.Axis', () => {
+  var domElement: JQuery = angular.element(document.body).append('<div></div>');
+  var axis: n3Charts.Factory.Axis = undefined;
+  var eventMgr: n3Charts.Utils.EventManager = undefined;
+  var factoryMgr: n3Charts.Utils.FactoryManager = undefined;
```

Comment:
my idea was somehow, that we are testing each component for itself in the unit tests; if we arrange the tests clever we never have to define `eventMgr` and `factoryMgr`. Everything with more than 1 component should be tested in e2e tests. What do you think? Maybe we can test the `lifecycle` directly on the BaseFactory.